packages/console/src/components/Pagination/Next.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const Next = () => {
+  return (
+    <svg width="10" height="16" viewBox="0 0 10 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <path
+        d="M2.05417 15.0708L9.125 7.99997L2.05417 0.929134L0.875001 2.10747L6.76833 7.99997L0.875001 13.8925L2.05417 15.0708Z"
+        fill="#747778"
```

Comment:
We should remove `fill` property from SVGs, in case we would like to overwrite it with CSS.

Reply:
same as above, we have not aligned the "fill" usage, "fill" is seen in many place, I created LOG-2028 to track and discuss.